en/0-intro.md

```diff
@@ -2,38 +2,33 @@
 
 ## Abstract
 
-Today’s work market lacks efficiency because of high transaction costs[^1][^2][^3] and frequent conflicts of interests between the employer and the employee.
+The current work market is inefficient due to high transaction costs and persistent conflicts of interest between employers and employees. 
 
-The WoM (World of Meeds) proposes a new paradigm for reducing transaction costs and reinventing working relationships. It bridges the gap between traditional companies and new forms of organisation by proposing a fair and transparent incentive system that realigns the interests of all parties.
+The World of Meeds (WoM) proposes a new paradigm to address these issues by reducing transaction costs and reinventing working relationships. It's a digital space that bridges the gap between traditional companies and new forms of organization by bringing a fair and transparent incentive system that realigns the interests of all parties.
 
 ## Introduction
 
-The work market today is broken. It fails to satisfy both workers and employers.
+The current state of the work market fails to satisfy both workers and employers[^1][^2][^3]. 
```

Comment:
```suggestion
The current state of the work market fails to satisfy both employees and employers concurrently [^1][^2][^3]. 
```

Reply:
I feel that the context will sound more corporate with the word "employees" rather than workers and "concurrently" will add more sense to the it

Reply:
a reason why we are making this change is to look less corporate.